<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadAvatar</class>
 <widget class="QWidget" name="UploadAvatar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cooing</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Take a picture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceList">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="captureBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="avatar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="cameraDisplayWidget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="retryBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Try again</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>submitBtn</zorder>
  <zorder>deviceList</zorder>
  <zorder>captureBtn</zorder>
  <zorder>cameraDisplayWidget</zorder>
  <zorder>retryBtn</zorder>
  <zorder>avatar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
